### DISCUSSION TITLE
username.github.io from privet repo

### DISCUSSION THREAD

**Discussion by Nikeshbajaj** [community/community#42888] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I had my page as **nikeshbajaj.github.io** deployed while ago, even though **nikeshbajaj.github.io** repository was privet, the site https://nikeshbajaj.github.io was accessible everywhere, however, any update on the pages  were note being reflected. So I made the repo public, which created deployment action, and all the changes were reflected to site. But now, if I make repo privet, site is not more accessible. 

I think, the simple question is:  How can I access the site, and still keeping the repository privet? (as it was before)

Thanks

**Comment by yoannchaudet** [comment]
Publishing Pages site from a private repo is a Pro feature. I assume you eventually went from Pro to Free which is why your site stopped building.
